pkg/components/helpers.go

```diff
@@ -227,3 +227,10 @@ func AddInitContainersToPodSpec(initContainers []string, podSpec *corev1.PodSpec
 	podSpec.InitContainers = append(containers, podSpec.InitContainers...)
 	return nil
 }
+
+func SelectTrueImage(coreImage string, instanceImage *string) string {
```

Comment:
suppose it can be non-exported since we are using it in the components package only

Reply:
And the name TrueImage sounds like some soft for windows. Let's maybe call It `SelectEffectiveImage`.
Also, I would expect a signature like `getImageWithDefault(componentImage, defaultImage string)`